src/Catalyst.Core.Modules.Ledger.Tests/IntegrationTests/LedgerKvmTests.cs

```diff
@@ -125,7 +144,8 @@ private void RunDeltas(Delta delta)
             _deltaHashProvider.DeltaHashUpdates.Returns(updates.Select(h => (Cid) h).ToObservable(_testScheduler));
 
             // do not remove - it registers with observable so there is a reference to this object held until the test is ended
-            var classUnderTest = new Ledger(_deltaExecutor, _stateProvider, _storageProvider, _stateDb, _codeDb, _fakeRepository, _deltaHashProvider, _ledgerSynchroniser, _mempool, _logger);
+            var classUnderTest = new Ledger(_deltaExecutor, _stateProvider, _storageProvider, _stateDb, _codeDb,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'classUnderTest'.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4562326)